Formula/liquigraph.rb

```diff
@@ -0,0 +1,23 @@
+class Liquigraph < Formula
+  desc "Migration runner for Neo4j"
+  homepage "http://www.liquigraph.org"
+  url "https://github.com/fbiville/liquigraph/archive/liquigraph-1.0-RC3.tar.gz"
+  sha256 "9b616bd8d8798374d316decd7c1ef60882bf81f6627511af4e3c710143882c47"
+  head "https://github.com/fbiville/liquigraph.git"
+
+  depends_on "maven" => :build
+  depends_on :java => "1.7+"
+
+  def install
+    system "mvn", "-q", "clean", "package", "-DskipTests"
```

Comment:
Yup, you got it in one.

We have a build/test sandbox that is enabled on the CI _(and will eventually be enabled by default for users)_ so things can't write to `$HOME` during the build or test process _(Doesn't impact or modify behaviour of the tool after installation)_, which is what Maven attempts to do automatically. `ENV.java_cache` just tells it to use `HOMEBREW_CACHE` instead.